plugins/arUnogPreservicaPlugin/README

```diff
@@ -0,0 +1,89 @@
+UNOG Preservica Plugin
```

Comment:
Thanks for this README! It was really helpful for getting my head around what's being done in this plugin :1st_place_medal: